### PR TITLE
9.0 introspection

### PR DESCRIPTION
**Pull request by nhomar**
- Better views.
- Proper instrospection management of instances.
- Tested with new vauxooci.
- [x] Lints.
- [ ] Unittests.

### CODE REVIEW

**Comment by nhomar** [comment]
@moylop260 Can you give me your +1 please.

File: runbot_travis2docker/views/runbot_repo_view.xml [hunk 1]
```diff
@@ -2,17 +2,67 @@
...
+		</xpath>
+        <xpath expr="//group" position="before">
+			<label for="name" class="oe_edit_only"/>
+			<h1><field name="name"/></h1>
+			<label for="group_ids" class="oe_edit_only"/>
+			<field name="group_ids" widget="many2many_tags"/>
+		</xpath>
+        <xpath expr="//group" position="replace">
+			<group>
+				<group name="parameters" cols="3" string="Paramenters">
```
Comment (moylop260) [inline runbot_travis2docker/views/runbot_repo_view.xml:53]: s/Paramenters/Parameters/g

File: runbot_travis2docker/views/runbot_repo_view.xml [hunk 2]
```diff
@@ -2,17 +2,67 @@
...
+					<field name="mode"/>
+					<field name="nginx"/>
+					<field name="token"/>
+					<field name="hook_time" readonly="1"/>
+				</group>
+				<group name="travis2docker" cols="3" string="Travis2Docker Paramenters">
```
Comment (moylop260) [inline runbot_travis2docker/views/runbot_repo_view.xml:59]: s/Paramenters/Parameters/g

**Comment by moylop260** [comment]
The red of travis is unrelated because show a warning:
screen shot 2016-05-16 at 3 49 37 pm

``` bash
2016-05-16 20:20:44,659 141 WARNING openerp_test openerp.addons.base.res.res_lang: Unable to get information for locale fr_FR. Information from the default locale (en_US) have been used.
```

This error was reported here: https://github.com/Vauxoo/docker-odoo-image/issues/118

**Comment by moylop260** [comment]
I have created a t2d instance with this PR and I saw the Introspection field from runbot.build result: 
`"status": "failV", "url": "http://8362d2ca48dd:2002./instance_introspection.json", "error": "Failed to parse: 8362d2ca48dd:2002."}`

See image below
instrospection

**Comment by moylop260** [comment]
I can't to reproduce the pylint mqt error in t2d yet... but the output was:

``` bash
======== Testing test_flake8 ========
runbot_travis2docker/models/runbot_build.py:57:80: E501 line too long (80 > 79 characters)
runbot_travis2docker/models/runbot_build.py:59:80: E501 line too long (102 > 79 characters)
runbot_travis2docker/models/runbot_build.py:61:80: E501 line too long (81 > 79 characters)
runbot_travis2docker/models/runbot_build.py:64:17: E128 continuation line under-indented for visual indent
runbot_travis2docker/models/runbot_build.py:70:80: E501 line too long (80 > 79 characters)
runbot_travis2docker/models/runbot_build.py:72:80: E501 line too long (84 > 79 characters)
runbot_travis2docker/models/runbot_build.py:76:39: E201 whitespace after '{'
runbot_travis2docker/models/runbot_build.py:76:80: E501 line too long (88 > 79 characters)
runbot_travis2docker/models/runbot_build.py:80:80: E501 line too long (156 > 79 characters)
runbot_travis2docker/models/runbot_build.py:85:80: E501 line too long (144 > 79 characters)
runbot_travis2docker/models/runbot_build.py:101:34: E127 continuation line over-indented for visual indent
runbot_travis2docker/models/runbot_build.py:102:80: E501 line too long (93 > 79 characters)
runbot_travis2docker/models/runbot_repo.py:12:80: E501 line too long (83 > 79 characters)
======== Testing test_pylint ========
extra_params_cmd ['--extra-params', '--load-plugins=pylint_odoo']
************* Module runbot_travis2docker.models.runbot_build
runbot_travis2docker/models/runbot_build.py:62: [W0703(broad-except), RunbotBuild._get_introspection] Catching too general exception Exception
runbot_travis2docker/models/runbot_build.py:75: [W0703(broad-except), RunbotBuild._get_introspection_html] Catching too general exception Exception
```

**Comment by nhomar** [comment]
I Knosw what is the problem.

But if you can simply review deeper if something is wrong into the .travis it should be good.

**Comment by moylop260** [comment]
The error of pylint is because we are using `sudo: required` for use docker then show this error in this version of ubuntu.

I have created a new issue for this case:
https://github.com/Vauxoo/pylint-odoo/issues/31

NOTE: I can't reproduce it with travis2docker because that is a `sudo: false` environment.

**Comment by nhomar** [comment]
@moylop260 Please check the error on this PR:

http://screenshots.vauxoo.com/oem/d5818b-1180x577.png

I do not think it is related to this PR itself, can you check it out?.

**Comment by moylop260** [comment]
You are right . it is unrelated.
https://github.com/Vauxoo/maintainer-quality-tools/issues/109

**Pull request merged**